packages/account/package.json

```diff
@@ -33,7 +33,7 @@
         "@deriv-com/analytics": "1.4.13",
         "@deriv/api-types": "^1.0.172",
         "@deriv/api": "^1.0.0",
-        "@deriv-com/quill-ui": "^1.9.22",
+        "@deriv-com/quill-ui": "^1.9.24",
```

Comment:
should'nt we have changes to package-lock json file?
can you run npm run bootstrap: dev